TombEngine/Specific/Video/Video.cpp

```diff
@@ -377,8 +376,18 @@ namespace TEN::Video
 		// Attempt to map and render texture only if callback has set frame to be rendered.
 		if (_needRender)
 		{
-			auto mappedResource = D3D11_MAPPED_SUBRESOURCE{};
-			if (_videoTexture && SUCCEEDED(_d3dContext->Map(_videoTexture, 0, D3D11_MAP_WRITE_DISCARD, 0, &mappedResource)))
+			_device->UpdateTexture2D(_videoTexture.get(), _frameBuffer);
+
+			if (_playbackMode == VideoPlaybackMode::Exclusive)
+			{
+				RenderExclusive();
+			}
+			else if (_playbackMode == VideoPlaybackMode::Background)
+			{
+				RenderBackground();
+			}
+
+			/*if (_videoTexture && SUCCEEDED(_d3dContext->Map(_videoTexture, 0, D3D11_MAP_WRITE_DISCARD, 0, &mappedResource)))
```

Comment:
Why this block is commented? Was the video texture tested after this was removed?